cmd/admin-handlers.go

```diff
@@ -1263,10 +1263,6 @@ func (a adminAPIHandlers) ObjectSpeedTestHandler(w http.ResponseWriter, r *http.
 		}
 	}
 
-	defer objectAPI.DeleteObject(ctx, customBucket, speedTest+SlashSeparator, ObjectOptions{
-		DeletePrefix: true,
-	})
-
```

Comment:
We should still remove data on this @vadmeste current code is correct, not sure why are you removing this. 

If you wish to preserve the generated data you must ask for another query param or http header for this API

Reply:
> If you wish to preserve the generated data you must ask for another query param or http header for this API

Yes, I can do that as well.